### PR TITLE
Add OceanColor

### PR DESCRIPTION
**Pull request by castelao**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
-  [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@conda-forge/help-python , ready for review

### CODE REVIEW

File: recipes/OceanColor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - OceanColor=OceanColor.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - click >=7.1.1 numpy>=1.17 h5netcdf>=0.8 h5py>=2.10 tables>=3.6 netcdf4>=1.5 pandas>=1.0 pyproj>=2.6 requests>=2.23 xarray>=0.16
```
Comment (dopplershift) [inline recipes/OceanColor/meta.yaml:26]: Looks like something left over?
```suggestion
    - click >=7.1.1
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just need to fix the one line and this is ready to go.

**Comment by dopplershift** [comment]
Assuming CI passes, that is.

File: recipes/OceanColor/meta.yaml (hunk 1 above)
Comment (castelao) [reply recipes/OceanColor/meta.yaml]: Ooops. Thanks Ryan!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**